DataStructures.Tests/Heap/MinMaxHeapTests.cs

```diff
@@ -152,6 +163,7 @@ public static void GetMinTest<T>(IEnumerable<T> collection)
             extracted.Add(value);
         }
 
-        Assert.IsTrue(ordered.SequenceEqual(extracted));
+        Assert.That(ordered.SequenceEqual(extracted), Is.True);
     }
+    #endregion
```

Comment:
I don't think you need regions here any more because now they only have 1 test in them